transactions/README.md

```diff
@@ -97,36 +107,69 @@ fileName:transaction.json
 }</code><span class="line-numbers-rows" aria-hidden="true"><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span></span></pre></div>
 </details>
 
-There are two types of transactions: authorisation transactions and clearing transactions. **Authorisation transactions** are processed when the customer is making the payment in-store or online. They trigger the `transaction.auth` webhook event that you can use to notify or reward the user in your application in real-time.
+The Fidel API currently supports three types of transactions: authorisation transactions, clearing transactions and refund transactions.
+
+**Authorisation transactions** are processed when a purchase registers on a linked card. For example, when a customer is making a payment in-store or online in real-time. When a customer makes a payment with a linked debit/credit card in an auth-enabled location, the `transaction.auth` webhook is also triggered and the transaction object sent to your specified URL in real-time.
+
+**Clearing transactions**, also known as "settled transactions", are processed when a payment transaction settles, usually happens 48 to 72 hours after a payment registers. The Fidel processes for clearing transactions are also triggering the `transaction.clearing` webhook events. The processes run daily at 12:00 UTC for Mastercard and multiple times per day for Visa and American Express. Only one transaction is sent per event.
```

Comment:
can you use smartypants quotes please? 
‘single quotes’
left: option + ], right: option + shift + ]
“double quotes”
left: option + [, right: option + shift + ]

the easier solutions: 
a) i think if you mix these with "regular" quotes in grammarly, gm will ask you to choose one style.
b) some writers, like Bear, automagically makes quotes smart (quite annoyingly in code blocks, too 🤦‍♀️ )